tests/test_vxlan_vtep.rb

```diff
@@ -202,6 +202,9 @@ def test_source_interface
   end
 
   def test_source_interface_hold_down_time
+    skip('source_intf_hold_down_time not supported on this Platform') unless
+    node.cmd_ref.supports?('vxlan_vtep', 'source_intf_hold_down_time')
+
```

Comment:
[`validate_property_excluded?` does the same check](https://github.com/cisco/cisco-network-node-utils/blob/develop/tests/ciscotest.rb#L142-L144). Please remove this skip and use the `validate_property_excluded` method instead